app/swagger/requests/user.rb

```diff
@@ -99,6 +99,8 @@ class User
                       key :type, :string
                     end
                   end
+                  property :va_patient, type: :boolean
+                  property :mhv_account_state, type: :string, enum: %w[OK DEACTIVATED MULTIPLE NONE], example: 'OK'
```

Comment:
Thoughts on adding a `description:`? `MULTIPLE` & `NONE` in particular don't seem immediately obvious to me what they mean.